packages/config/config/devices/0x027a/zen23_4.0.json

```diff
@@ -0,0 +1,238 @@
+// Zooz ZEN23
+// Z-Wave Plus On/Off Toggle Switch v4
+{
+	"revision": 1,
```

Comment:
You can remove this - we've opted not to import OZW files, so this is no longer necessary.